app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -92,8 +94,8 @@ private boolean isCandyWordFirst() {
      * @param simpleBG     Whether the bitmap has a simple background
      * @return Bitmap with replaced colors
      */
-    private Bitmap replaceColors(Bitmap srcBitmap, int keepCr, int keepCg, int keepCb, int replaceColor, int distance,
-                                 boolean simpleBG) {
+    private Bitmap replaceColors(Bitmap srcBitmap, Bitmap dstBitmap, int keepCr, int keepCg, int keepCb,
```

Comment:
Good point: the API I'm using allows `dstBitmap` to be `!= null` and `!= srcBitmap`, but we're not using that.